site/app/controllers/admin/UsersController.php

```diff
@@ -269,9 +269,9 @@ public function updateUser($type = 'users') {
             }
         }
         //Pronouns must be less than 12 characters.
-        // if (!empty(trim($_POST['user_pronouns']))) {
-        //     $error_message .= User::validateUserData('user_pronouns', trim($_POST['user_pronouns'])) ? "" : "Error in pronouns: \"" . strip_tags($_POST['user_pronouns']) . "\"<br>";
-        // }
+        if (!empty(trim($_POST['user_pronouns']))) {
+            $error_message .= User::validateUserData('user_pronouns', trim($_POST['user_pronouns'])) ? "" : "Error in pronouns: \"" . strip_tags($_POST['user_pronouns']) . "\"<br>";
```

Comment:
Why not move the condition `User::validateUserData('user_pronouns', trim($_POST['user_pronouns']))` into the `if`-statement above instead of what is essentially two nested `if`-statements?

Reply:
fixed